packages/backend/src/services/InstanceConfigurationService/InstanceConfigurationService.ts

```diff
@@ -197,7 +197,7 @@ export class InstanceConfigurationService extends BaseService {
                 const allowedDomains: AllowedEmailDomains = {
                     organizationUuid,
                     emailDomains,
-                    role: OrganizationMemberRole.VIEWER,
+                    role: setup.organization.defaultRole,
```

Comment:
it seems we were not initializing the role using the variable :facepalm: Fixed here